app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,17 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(self,
+                 nickname: str,
+                 musical_instrument: str,
+                 bow_level: int) -> None:
```

Comment:
It is a better practice to place each argument in the `__init__` on a new line to improve the readability. There should be no arguments on the same line with opening and closing parenthesis. Please, change your code style in all classes to encounter that.